filters/tee/tee.go

```diff
@@ -171,6 +172,11 @@ func (r *tee) Request(fc filters.FilterContext) {
 	}()
 }
 
+func (r *tee) Close() error {
+	r.client.Close()
+	return nil
+}
```

Comment:
Note that this is not called so we'll leak http clients. 
For the auth filters we mitigate this by keeping a global client cache of clients per url.

Reply:
True this requires a fix before merge

Reply:
fixed